vraptor-core/src/main/java/br/com/caelum/vraptor/http/iogi/IogiParametersProvider.java

```diff
@@ -90,11 +90,12 @@ private Object instantiateOrAddError(Parameters parameters, List<Message> errors
 	private List<Target<Object>> createTargets(ControllerMethod method) {
 		Method javaMethod = method.getMethod();
 		List<Target<Object>> targets = new ArrayList<>();
+		TypeExtractor typeExtractor = new TypeExtractor(method.getController().getType());
```

Comment:
When I created the TypeExtractor class, my intention was to allow this feature to be reused if there was a need.
Thinking about the opinion of @garcia-jj, it makes sense because VRaptor exists a few years ago and yet there was no need to use this feature.
Maybe have less impact on the project if we keep this feature in their own class (private method).
@Turini, What do you think?

** Sorry for my terrible english. :-/